From 5b13e07c0a672de74dfa5986f108c3c6b6e9324b Mon Sep 17 00:00:00 2001
From: Eliad Peller <eliad@wizery.com>
Date: Thu, 12 Dec 2013 15:52:15 +0200
Subject: [PATCH] undef KEY_FILE if defined

some kernels define KEY_FILE somewhere else (auto-included),
which cause a conflict. simply undef it in this case.

Signed-off-by: Eliad Peller <eliad@wizery.com>
---
 net/mac80211/debugfs_key.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/net/mac80211/debugfs_key.c b/net/mac80211/debugfs_key.c
index 1521cab..b50fdc7 100644
--- a/net/mac80211/debugfs_key.c
+++ b/net/mac80211/debugfs_key.c
@@ -34,6 +34,10 @@ static const struct file_operations key_ ##name## _ops = {		\
 	.llseek = generic_file_llseek,					\
 }
 
+#ifdef KEY_FILE
+#undef KEY_FILE
+#endif
+
 #define KEY_FILE(name, format)						\
 		 KEY_READ_##format(name)				\
 		 KEY_OPS(name)
-- 
1.8.5.rc1

